tools/FileTestDefs.sh

```diff
@@ -413,6 +413,29 @@ namespace_details_test() {
 }
 standard_checks+=(namespace_details)
 
+# Check for .cpp includes in headers and tpp files
+cpp_includes_in_includibles() {
+    is_c++ "$1" && staged_grep -q "include .*\.cpp" $1
```

Comment:
Quote second `$1`.

Reply:
For my own understanding, what is the difference in meaning between the quoted and unquoted version?

Reply:
The unquoted version does word splitting, so if `$1` is `Hello world` then `func "$1"` is the same as `func "Hello world"` and `func $1` is the same as `func "Hello" "world"`.  It doesn't really matter here since I'm sure lots of other things will be unhappy if we have filenames containing whitespace, but this file has pretty good bash style overall so I'm being slightly picky about it.

Reply:
Ah okay, that makes sense :) I appreciate you teaching me the bash ways :)